addons/test_mass_mailing/models/mailing_models.py

```diff
@@ -78,3 +78,13 @@ class MailingPerformanceBL(models.Model):
     container_id = fields.Many2one(
         'mail.test.container', 'Meta Container Record',
         tracking=True)
+
+class MailingTestPartner(models.Model):
```

Comment:
Maybe to defined after "mailing.test.optout", as models defined afterwards are performance models ?